### ISSUE TITLE
An Illegal instruction (core dumped) Error

### ISSUE THREAD

**Issue by yangchunmian123** [osu-crypto/libPSI#62]
Hi Professor, I'm having the following problem with my unit tests after compiling, I can't figure out why this is happening, can you please help me?

![image](https://user-images.githubusercontent.com/118784421/227493998-571e3b00-b6da-4a4e-8f5e-c7efdfc942a3.png)

Also, I thought that a problem with a tool might not affect the operation of certain protocols, so I ran the following command and got the following error.

![image](https://user-images.githubusercontent.com/118784421/227496461-f7434823-633f-42aa-bc2a-9cc6a33307ac.png)

I don't know what's wrong, do you have any ideas please?

**Comment by ladnir** [comment]
Bitpolymul requires CPUs with sse2. It appears your system does not have this.

**Comment by yangchunmian123** [comment]
Thank you very much, Professor, this is a question I hadn't thought of. But I checked the CPU model of my device which has the sse2 instruction set.

![image](https://user-images.githubusercontent.com/118784421/227720354-1d7ef9fc-b503-4eb5-85b6-669313ca50ee.png)

This has given me new ideas to solve the problem, but I don't know what instruction sets are required for this project. Also, I was able to successfully compile the libOTe project but was unable to unit test it and then reported the same error, so I don't know if this information would be helpful in troubleshooting the problem.

![image](https://user-images.githubusercontent.com/118784421/227720685-4efc5002-a775-47d7-8254-365cd389363f.png)
